client/src/containers/Dog/DogInfo.js

```diff
@@ -0,0 +1,45 @@
+// import {
+//     Card,
+//     CardContent,
+//     CardMedia,
+//     makeStyles,
+//     Typography,
+//   } from "@material-ui/core";
```

Comment:
Better to remove any used imports instead of commenting

Reply:
removed in latest commit